.circleci/rebuild.hs

```diff
@@ -58,6 +58,57 @@ skipAlgo =
   ]
 
 
+skipCfx :: [FilePath]
+skipCfx =
+  [ "tut-7-rpc" -- TODO: test rpc w/ cfx
+  , "tut-8" -- TODO: add cfx to tut-8
+
+  -- constructor args not yet supported
+  --  at ContractFactory.deploy (cfxers)
+  , "atomic-swap"
+  , "atomic-swap-auction"
+  , "chicken-fork"
+  , "chicken-parallel"
+  , "chicken-race"
+  , "race"
+  , "weird-swap"
+  , "workshop-hash-lock"
+  , "workshop-relay"
+  , "workshop-trust-fund"
+
+  -- Tx with same nonce already inserted.
```

Comment:
My guess is that these errors are okay because it is just a sign that these participants are competing to be the next publisher